index.html

```diff
@@ -128,6 +128,12 @@
                             <li v-if="'live-initramfs' in build.meta.images">
                                 Live Initramfs: <span v-html="getImageUrl(build, 'live-initramfs')"></span>
                             </li>
+                            <li v-if="'gcp' in build.meta.images">
+                                GCP: <span v-html="getImageUrl(build, 'gcp')"></span>
+                            </li>
+                            <li v-if="'azure' in build.meta.images">
```

Comment:
Hmm, we already have Azure up there, right?

Reply:
right this was an issue created when I rebased my local patch set